targets/PythonSdk/source/source/PlayFabHTTP.py

```diff
@@ -0,0 +1,68 @@
+import json
+import PlayFabSettings
+import PlayFabErrors
+import requests
+
+# Note this is a blocking call and will always run synchronously
+# the return type is a dictionary that should contain a valid dictionary that
+# should reflect the expected JSON response
+# if the call fails, there will be a returned PlayFabError
+def DoPost(urlPath, request, authKey, authVal, extraHeaders, callback):
+    url = PlayFabSettings.GetURL(urlPath)
```

Comment:
It's a trivial optimization, but we can move the call to GetURL after the try below.